src/card/dto/response-card.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { CreateCardDto } from './create-card.dto';
+import { PartialType } from '@nestjs/mapped-types';
+import {
+    IsNotEmpty,
+    IsString,
+} from 'class-validator';
+
+export class ResponseCardDto extends PartialType(CreateCardDto) {
+    @IsString()
+    @IsNotEmpty()
+    id: string;
+
+    constructor(id: string, name: string) {
```

Comment:
I wonder, if this constructor can be in fields definition

Reply:
See later